src/CoreWCF.Primitives/src/CoreWCF/IdentityModel/Tokens/UserNameSecurityToken.cs

```diff
@@ -11,12 +11,10 @@ internal class UserNameSecurityToken : SecurityToken
     {
         private readonly string _id;
         private readonly DateTime _effectiveTime;
-
         public UserNameSecurityToken(string userName, string password)
```

Comment:
You've removed blank lines before and after the constructor which should be there.